cookbooks/Berksfile

```diff
@@ -6,4 +6,7 @@ cookbook 'apt', '~> 2.3.8'
 cookbook 'build-essential', '~> 2.1.3'
 cookbook 'chef-client', '~> 4.2.0'
 cookbook 'ntp', '~> 1.8.6'
-cookbook 'brightbox-ruby', '~> 1.2.1'
+
+Dir.entries('.').select{|x| File.directory?(x) && !x.start_with?('.')}.each do |path|
+  cookbook(File.basename(path), path: path)
```

Comment:
This seems like it could be fragile; it looks like there are many non-cookbook files in the cookbook directory (Berkfile, Gemfile, chefignore, etc).  Are these (or future files that might be added to this directory) correctly treated as non-cookbooks?

Would it be a bad idea to explicitly list the cookbooks in the Berkfile?